HW19/js/script.js

```diff
@@ -0,0 +1,65 @@
+ var counter = 0, sum = 0;
+
```

Comment:
I don't see any update to use the `local storage` !!!

Reply:
I created a new repo (hw20)  where the updated code 
And attached new CSS file which differs from  the CSS file that is already included in hw19.